addons/containerd/1.6.33/install.sh

```diff
@@ -195,6 +195,11 @@ EOF
         "$DIR/bin/toml" -basefile=/etc/containerd/config.toml -patchfile="$tmp"
     fi
 
+    if is_ubuntu_2404 ; then
+        # we need to disable apparmor on ubuntu 24.04 to allow pods to be deleted
+        sed -i 's/disable_apparmor = false/disable_apparmor = true/' /etc/containerd/config.toml
+    fi
```

Comment:
I do not LIKE this solution
I think there is probably a better solution
I will keep looking for a better solution
This allows installations to complete in my testing